Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -716,7 +716,7 @@ theorem Monotone.tendsto_le_alternating_series
       ← sub_eq_add_neg, sub_le_iff_le_add]
     gcongr
     exact hfm (by omega)
-  exact ha.le_of_tendsto (hfl.comp (tendsto_atTop_mono (fun n ↦ by omega) tendsto_id)) _
+  exact ha.le_of_tendsto (hfl.comp (tendsto_atTop_mono (fun n ↦ by dsimp; omega) tendsto_id)) _
```

Comment:
I'm happy with this fix, just flagging that it isn't the `set_option` variant that we see almost everywhere else.

Reply:
We decide that `omega` can't afford to do identification of atoms up to defeq: in some problems it's too expensive.

So slightly more preprocessing will be needed in some cases.